pokedex/simulator/include/pokedex_interface.h

```diff
@@ -55,6 +55,9 @@ struct pokedex_model_description {
     // informationaly only
     const char* model_isa;
 
+    // supported privilege string.
+    const char model_priv[3];
+
```

Comment:
`const char* model_priv` is OK, using `char char []` is much more tricky, e.g. "MSU" is actually 4 byte. Since typically model desc is only parsed once in creation, no bother for the complexity here.